kentikapi/client.go

```diff
@@ -3,15 +3,22 @@ package kentikapi
 import (
 	"time"
 
+	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/cloudexport"
 	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/httputil"
+	"github.com/kentik/community_sdk_golang/apiv6/kentikapi/synthetics"
 	"github.com/kentik/community_sdk_golang/kentikapi/internal/api_connection"
 	"github.com/kentik/community_sdk_golang/kentikapi/internal/resources"
 )
 
 // Public constants.
+//nolint:gosec
 const (
-	APIURLUS = "https://api.kentik.com/api/v5"
-	APIURLEU = "https://api.kentik.eu/api/v5"
+	APIURLUS          = "https://api.kentik.com/api/v5"
```

Comment:
Please put public constants in separate _const_ block with informative comment, e.g. _Kentik API URLs_

Reply:
"// Public constants" comment no longer applies.